index.js

```diff
@@ -21,6 +21,8 @@ module.exports = function(str, options) {
 
   var newline = options.newline || '\n' + indent;
 
+  var escape = options.escape || function(str){return str;};
```

Comment:
would you mind changing this to:

```js
function identity(str) { 
  return str; 
};
var escape = typeof options.escape === 'function'
  ? options.escape
  : identity;
```
so we can explicitly ensure that `escape` will always be a function (and not `true` or something)